server/src/main/java/org/elasticsearch/inference/InputType.java

```diff
@@ -8,17 +8,12 @@
 
 package org.elasticsearch.inference;
 
-import org.elasticsearch.common.io.stream.StreamInput;
-import org.elasticsearch.common.io.stream.StreamOutput;
-import org.elasticsearch.common.io.stream.Writeable;
-
-import java.io.IOException;
 import java.util.Locale;
 
 /**
  * Defines the type of request, whether the request is to ingest a document or search for a document.
  */
-public enum InputType implements Writeable {
```

Comment:
I decided to remove these because the cohere task settings need to write it as an optional enum so I figured it'd be better to just leave it up to the caller to handle how it is written.